packages/terra-popup/src/Popup.scss

```diff
@@ -0,0 +1,5 @@
+@import './variables';
+
```

Comment:
This one can't be, unfortunately as it's the class prefix passed to the tether component, and we have no control over tethers implementation.